peepingtom/io_/read/star/reader_functions.py

```diff
@@ -49,7 +52,7 @@ def extract_data(df, mode='3.1', name_regex=None):
 
         coords = df_volume[coord_headings[dim]].to_numpy(dtype=float)
         shifts = df_volume.get(shift_headings[dim][mode], pd.Series([0.0])).to_numpy()
-        px_size = df_volume.get(pixel_size_headings, pd.Series([1.0])).to_numpy()
+        px_size = df_volume.get(pixel_size_headings[mode], pd.Series([1.0])).to_numpy()
```

Comment:
do rln 3.0 files have anything which tells you pixel size in the file? I didn't think they did